docker/ci/centos7.9.2009/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM centos:centos7.9.2009
+
+# fix repos for EOL centos
+ADD https://raw.githubusercontent.com/AtlasGondal/centos7-eol-repo-fix/main/CentOS-Base.repo /etc/yum.repos.d/
```

Comment:
This file feels a file we ought to have local to our own repo, especially because we already have another similar file.

Reply:
I couldn't decide on whether to vendor or fetch, but your comment decides it.  I *AM* going to leave the `ADD` with the URL in the dockerfile as a durable comment indicating the source of that file (unless you think something else would be better).